R/include-hba1c.R

```diff
@@ -1,32 +1,45 @@
 #' Include only those with HbA1c in the required range.
```

Comment:
Would it be more correct to say this instead?
```suggestion
#' Include only those with HbA1c above the required threshold.
```